build.sbt

```diff
@@ -51,6 +51,9 @@ lazy val keepExistingHeader =
         .trim()
   )
 
+ThisBuild / versionScheme := Some("early-semver")
+ThisBuild / versionPolicyIntention := Compatibility.BinaryAndSourceCompatible
```

Comment:
Is this doing the MiMa check like we do in scio ?

Reply:
Yes, plugin `"ch.epfl.scala" % "sbt-version-policy" % "2.1.0"` does MiMa check